django/contrib/admin/templates/registration/logged_out.html

```diff
@@ -1,7 +1,7 @@
 {% extends "admin/base_site.html" %}
 {% load i18n %}
 
-{% block breadcrumbs %}<div class="breadcrumbs"><a href="{% url 'admin:index' %}">{% translate 'Home' %}</a></div>{% endblock %}
+{% block breadcrumbs %}<div class="breadcrumbs"><a href="{% url 'admin:index' %}" aria-current="page">{% translate 'Home' %}</a></div>{% endblock %}
```

Comment:
I'm not sure that we should add this. The logged out page is not the same as that link to the admin site home page...
```suggestion
{% block breadcrumbs %}<div class="breadcrumbs"><a href="{% url 'admin:index' %}">{% translate 'Home' %}</a></div>{% endblock %}
```